qiskit/unrollers/_circuitbackend.py

```diff
@@ -10,8 +10,7 @@
 """
 
 from qiskit import QuantumCircuit
-from ._backenderror import BackendError
-from ._unrollerbackend import UnrollerBackend
+from qiskit.unroll import BackendError, UnrollerBackend
```

Comment:
This looks like a rebase issue. The imports here should be either what was already there in L13-L14 or changed to be an absolute import from `qiskit.unrollers`. Importing from `qiskit.unroll` like this will cause pylint to complain about a circular import because the import path looks like `qiskit.unrollers._circuit_backend` -> `qiskit.unroll` -> `qiskit.unrollers._circuit_backend`.